src/search/pruning/stubborn_sets.h

```diff
@@ -94,16 +81,9 @@ class StubbornSets : public PruningMethod {
     virtual void initialize_stubborn_set(const State &state) = 0;
     virtual void handle_stubborn_operator(const State &state, int op_no) = 0;
 public:
-    explicit StubbornSets(const options::Options &opts);
+    StubbornSets();
```

Comment:
Not needed any more?

Reply:
Depending on if we want to manually initialize `num_operators` or not.